### ISSUE TITLE
Error work is not a valid argument for gam update user

### ISSUE THREAD

**Issue by anon238** [GAM-team/GAM#1586]
The issue tracker is for reporting product deficiencies. "How do I?" questions should be posted to the discussion forum at https://groups.google.com/group/google-apps-manager. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GAM release from https://github.com/GAM-team/GAM/releases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/GAM-team/GAM/wiki

Full steps to reproduce the issue:
1. `gam update user $NewUser otheremail work $NewUserEmail`
2. `ERROR: work is not a valid argument for gam update user`

Expected outcome (what are you trying to do?): Updating the otheremail with label work. 

Actual outcome (what errors or bad behavior do you see instead?): `ERROR: work is not a valid argument for gam update user`

**Comment by taers232c** [comment]
Check that $NewUser is defined; if not, the command looks like this and you get the error.

$ gam update user otheremail work ***@***.***

ERROR: work is not a valid argument for "gam update user"

Ross
----
Ross Scroggs
***@***.***

**Comment by anon238** [comment]
Yeah this variable is defined. Before that it’s running `gam create user $NewUsername1 firstname $NewUserFirstname1 lastname $NewUserLastname1 changepassword on org /` perfectly fine then the next command is `Start-Sleep -seconds 10` and the last one is `gam update user $NewUsername1 otheremail work $NewUserWEmail`

**Comment by taers232c** [comment]
It's getting undefined somehow; but to solve the problem just add `otheremail work $NewUserWEmail` to the end of the create command.

**Comment by anon238** [comment]
I will test this in a bit. However the next command is `gam update user $NewUserName1 password resetlink work` can I use this command also in the create user command?

**Comment by taers232c** [comment]
What is `password resetlink work`?

Any valid option in update user is valid in create user.

**Comment by anon238** [comment]
When you create a new user in Workspace you have an option to send a resetlink to a provided email address. I try to do this in GAM too.

**Comment by taers232c** [comment]
It's  not available in GAM.

**Comment by anon238** [comment]
Damn!! Writing the script for nothing 🤣

**Comment by theanswerco** [comment]
Hi, I need to disable the general settings tab  for specific users. Preferibly using  gam. I looked for an option "gam  update user <user email> gmailsetting disable_general_settings true" I received an error "gam update user <user email> gmailsetting disable_general_settings true"

**Comment by taers232c** [comment]
There is no API support for disabling this tab, thus GAM can'r help.

Ross
----
Ross Scroggs
***@***.***
